src/vlinder/make_report.py

```diff
@@ -0,0 +1,234 @@
+"""
+This file contains two functions which make a report of the information performed in the tool.
+"""
+
+import os
+import shutil
+import cv2
+from pathlib import Path
+from datetime import datetime
+from vlinder.presentation import PDF
+
+
+
+class MakeReport:
+    """
+    This class deals with the transformation into a different format and export of output of an RBS case.
+    """
+
+    def __init__(self, output_path, name, input_dict, output_dict, visualize, random_number):
+        self.output_path = Path(output_path)
+        self.folder_name = ""
+        self.name = name
+        self.input_dict = input_dict
+        self.output_dict = output_dict
+        self.page_number = 1
+        self.visualize = visualize
+        self.random_number = random_number
+
+    def determine_position_images(self, orientation, image):
+        """
+        This function gives the possibility to place images at the right places which the desired orientation
+        :param orientation: the desired orientation for PDF format; there is a choice between Portrait of Landscape
+        :param image: information about the image
+        :return: the right position
+        """
+        if orientation == "Landscape":
+            max_image, width_image, x_pos = 800, 250, 150 - image.shape[1] / 5.32
+        else:
+            max_image, width_image, x_pos = 500, 150, 100 - image.shape[1] / 6.3
+        return max_image, width_image, x_pos
+
+    def make_title(self, target, scenario="", pos_series="", key_output="") -> str:
```

Comment:
`make_title`, `make_strategic_challenge` en `make_introduction` zouden een stuk korter kunnen (maar functionaliteit is goed verder)

e.g. make_title:

    text_element_key = f"title_{target}"
    text_element_type = "case_text_element" if text_element_key in self.input_dict["case_text_elements"] else "generic_text_element"
    
    if text_element_key in self.input_dict[f"{text_element_type}s"]:
        index = self.input_dict[f"{text_element_type}s"].index(text_element_key)
        text_value = self.input_dict[f"{text_element_type}_value"][index]
        if text_value != "nan":
            return text_value + scenario + pos_series + key_output
    
    return "Not defined in template"